python_modules/dagster/dagster/_utils/__init__.py

```diff
@@ -651,7 +651,7 @@ def process_is_alive(pid: int) -> bool:
 
 
 def compose(*args):
```

Comment:
does this even have any callers?

Reply:
Tried searching and looks like no. Probably a bunch of utils functions like that, though I would say it's for another PR to remove dead code.